pkg/metadata/vmmd/fips.go

```diff
@@ -0,0 +1,18 @@
+package vmmd
+
+import (
+	"os"
+)
+
+// FIPSEnabled returns true if running in FIPS mode.
+// currently it just checks the system wide /etc/system-fips file present or not.
+// We can improve it later.
+func FIPSEnabled() bool {
```

Comment:
Please move this to `pkg/util`, that seems like a better place for it.

Reply:
Fixed